include/nbl/video/IPhysicalDevice.h

```diff
@@ -76,6 +79,14 @@ class IPhysicalDevice : public core::Interface, public core::Unmovable
         bool shaderSubgroupQuadAllStages = false;
     };
 
+    struct SMemoryProperties
+    {
+        uint32_t        memoryTypeCount;
+        VkMemoryType    memoryTypes[VK_MAX_MEMORY_TYPES];
+        uint32_t        memoryHeapCount;
+        VkMemoryHeap    memoryHeaps[VK_MAX_MEMORY_HEAPS];
```

Comment:
hmmm... wonder what OpenGL should return here XD

Reply:
This is still a TODO on OpenGL.
Should I open an issue for it?

Reply:
its not a big task, lets just decide what makes sense here.

Shall we just set the type and heap counts on OpenGL to 0 and call it a day?

Reply:
assigned to @Erfan-Ahmadi , lets set the counts to 0 and call it a day.

Reply:
obsolete, choosing to spoof.